src/main/kotlin/org/team5419/fault/math/geometry/Rotation2d.kt

```diff
@@ -1,120 +1,58 @@
 package org.team5419.fault.math.geometry
 
-import org.team5419.fault.math.Epsilon
+import org.team5419.fault.math.epsilonEquals
+import org.team5419.fault.math.kEpsilon
+import org.team5419.fault.math.units.SIUnit
+import org.team5419.fault.math.units.derived.*
```

Comment:
Isn't doing mass imports considered bad practice? I know are best friend Tox doesn't like it.